rust/src/editor.rs

```diff
@@ -198,7 +191,26 @@ impl Editor {
         }
     }
 
+    fn delete_forward(&mut self) {
+        if self.view.sel_start == self.view.sel_end {
+            self.move_right();
+        }
+
+        self.delete();
+    }
+
     fn delete_backward(&mut self) {
+        self.delete();
+    }
+
+    fn delete_to_beginning_of_line(&mut self) {
+        self.modify_selection();
+        self.move_to_left_end_of_line();
+
+        self.delete();
```

Comment:
On Note.app this behavior is wrong. But on "Sublime Text"  this behavior is right. How you think about it?